.travis/s2n_travis_build.sh

```diff
@@ -78,7 +78,12 @@ if [[ "$TESTS" == "ALL" || "$TESTS" == "integration" ]]; then make clean; make i
 if [[ "$TESTS" == "ALL" || "$TESTS" == "fuzz" ]]; then (make clean && make fuzz) ; fi
 if [[ "$TESTS" == "ALL" || "$TESTS" == "sawHMAC" ]] && [[ "$TRAVIS_OS_NAME" == "linux" ]]; then make -C tests/saw/ tmp/"verify_s2n_hmac_$SAW_HMAC_TEST".log ; fi
 if [[ "$TESTS" == "ALL" || "$TESTS" == "sawDRBG" ]]; then make -C tests/saw tmp/verify_drbg.log ; fi
-if [[ "$TESTS" == "ALL" || "$TESTS" == "tls" ]]; then make -C tests/saw tmp/verify_handshake.log ; fi
+if [[ "$TESTS" == "ALL" || "$TESTS" == "tls" ]]; then
+    # We have to output something every 9 minutes, as some proofs may run longer than 10 minutes
+    while sleep 9m; do echo "=====[ $SECONDS seconds still running ]====="; done &
+    make -C tests/saw tmp/verify_handshake.log
+    kill %1
+fi
```

Comment:
Have you checked to be sure this has the expected behavior with the pipefail? e.g. if the SAW proof fails does Travis see this command as failing? We had a problem with a line like line 78, without the subshell, the error codes weren't returned.

Reply:
The cleanup work you did on organization of the TLS Saw files is really nice

Reply:
I just intentionally broke one of the proofs, and it looks like make still fails, so Travis should see the command as failing:
```user error ("prove_handshake_io_lowlevel" (/home/ubuntu/s2n/tests/saw/verify_handshake.saw:25:1-25:28):
"crucible_llvm_verify" (/home/ubuntu/s2n/tests/saw/spec/handshake/handshake.saw:60:42-60:62):
Proof failed.)
Makefile:81: recipe for target 'tmp/verify_handshake.log' failed
make: *** [tmp/verify_handshake.log] Error 2
```

Reply:
I pushed a test commit that broke one of the proofs, and it looks like travis correctly fails: https://travis-ci.org/lrstewart/s2n/builds/614684066